website/docs/r/management_group.html.markdown

```diff
@@ -43,8 +43,12 @@ The following arguments are supported:
 
 * `group_id` - (Optional) The name or UUID for this Management Group, which needs to be unique across your tenant. A new UUID will be generated if not provided. Changing this forces a new resource to be created.
 
+~> **Note** `name` and `group_id` here corresponds to `ID` on Azure Portal.
```

Comment:
The Azure Portal changes all the time - this isn't something we should be relying on here



```suggestion
```

Reply:
Done